config/config_test.go

```diff
@@ -18,17 +17,22 @@ var repoMap = map[string]string{
 
 func TestLoad(t *testing.T) {
 	config, _ := Load(configYaml)
-	assert.Equal(t, repoMap, config.RepoMap)
+	if !reflect.DeepEqual(config.RepoMap, repoMap) {
```

Comment:
testify's `assert.Equal()` uses `reflect.DeepEqual()` under the hood. It successfully distinguishes e.g. `int` and `int64` at runtime:

```go
var foo int = 1
var bar int64 = 2
if !reflect.DeepEqual(foo, bar) {
	t.Fail()
}
```

However, if I use just a bare equality test, it fails at compile time:

```go
if foo != bar {
	t.Fail()
}
```

```
parser/parser_test.go:65: invalid operation: foo != bar (mismatched types int and int64)
```

Depending on perspective, the compile time failure is safer than the runtime check. After all, we're not writing ruby anymore... 😂 

Anyway, fewer deps for this simple project seems good to me. The biggest thing I miss out on is the diff on a failed equality assertion, but I'll worry about that later if it becomes a difficulty.